### ISSUE TITLE
Add polish and promote the Trustable Technology Mark page

### ISSUE THREAD

**Issue by xmatthewx** [MozillaFoundation/Design#309]
Let's add some polish to this page and then promote it on the parent initiatives page.

Page owner: Peter Bihr

 - [ ] add design polish, logo, etc
 - [ ] promote on initiatives page

Links
 - Page: https://foundation.mozilla.org/en/initiatives/trustable-technology-mark/
 - CMS: https://foundation.mozilla.org/cms/pages/242/edit/
 - Site: https://trustabletech.org

cc @kristinashu to triage

**Comment by beccaklam** [comment]
Design polish and logo added: https://foundation.mozilla.org/en/initiatives/trustable-technology-mark/

**Comment by kristinashu** [comment]
Sweet thanks @beccaklam! 

I'll work on adding it to the site.

@xmatthewx when you say Initiatives page, do you mean at the bottom of the page with `Featured Programs and Projects`?

**Comment by xmatthewx** [comment]
Yes and we should pass this along to Anil and Sydette to decide specifics for curation.
